containers/versions.yml

```diff
@@ -0,0 +1,7 @@
+debian_version:     "9.4"
+geth_version:       "v1.8.2"
+gunicorn_version:   "19.7.1"
+nginx_version:      "1.13.9-alpine"
+openresty_version:  "1.13.6.1-1-alpine-fat"
+postgres_version:   "10.3-alpine"
```

Comment:
I think that maybe it would be better to not include the `-alpine` part in the version and hard-code it in `Dockerfile`s instead. The `Dockerfiles` are built with the assumption that the underlying system is alpine so if someone changes it here they won't work.

Reply:
That's good idea, but I think we should also change variables e.g from `postgres_version` to `postgres_alpine_version`. When we will want some changes to version, we will not have to wonder what linux version of the image we are using.

Reply:
Done.